android/res/values/styles-text.xml

```diff
@@ -72,22 +72,21 @@
     <item name="android:textAllCaps">true</item>
   </style>
 
-  <style name="MwmTextAppearance.Toolbar"/>
-
-  <style name="MwmTextAppearance.Toolbar.Search">
+  <style name="MwmTextAppearance.Toolbar">
     <item name="android:textSize">@dimen/text_size_toolbar</item>
     <item name="android:textColor">@color/text_light</item>
+  </style>
+
+  <style name="MwmTextAppearance.Toolbar.Search">
     <item name="android:textColorHint">@color/text_light_hint</item>
   </style>
 
   <style name="MwmTextAppearance.Toolbar.Title" parent="@style/TextAppearance.Widget.AppCompat.Toolbar.Title">
-    <item name="android:textSize">@dimen/text_size_toolbar</item>
```

Comment:
Точно, спасибо. PTAL.
> On Mar 16, 2016, at 12:40 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/values/styles-text.xml <https://github.com/mapsme/omim/pull/2250#discussion_r56304656>:
> 
> >      <item name="android:textColorHint">@color/text_light_hint</item>
> >    </style>
> >  
> >    <style name="MwmTextAppearance.Toolbar.Title" parent="@style/TextAppearance.Widget.AppCompat.Toolbar.Title">
> > -    <item name="android:textSize">@dimen/text_size_toolbar</item>
> А вот здесь зря убрал? Обрати внимание, что парент у стиля другой
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2250/files/990db9858dbc453dee7bfb809aebd28b429d8f2d#r56304656>